packages/require/require.js

```diff
@@ -115,16 +115,12 @@ const Require = {
       networkType: options.networks[options.network].type
     });
 
-    try {
-      const fn = this.file({
-        file: options.file,
-        context: { web3, interfaceAdapter },
-        resolver: options.resolver
-      });
-      fn(done);
-    } catch (error) {
-      done(error);
-    }
+    const fn = this.file({
+      file: options.file,
+      context: { web3, interfaceAdapter },
+      resolver: options.resolver
+    });
+    await fn();
```

Comment:
One thing to note here, this would disallow using callbacks in the functions provided by users. Should we worry about this?

Reply:
We should preserve the ability for user scripts to be written with callbacks, cause this would break existing user code.

Reply:
So is the answer here to do something like
```
if (callbackPassed) {
  return fn(callback);
} else { 
  await fn()
}
```?